CODE/Logicytics.py

```
@@ -286,69 +288,93 @@
+                log.info(f"{EXECUTION_LIST[file]} executed")
         except UnicodeDecodeError as e:
             log.error(f"Error in code: {e}")
         except Exception as e:
             log.error(f"Error in code: {e}")
 
 
-@log.function
 def zip_generated_files():
     """Zips generated files based on the action."""
 
     def zip_and_log(directory, name):
```

Ruff (0.8.0)

Missing return type annotation for private function `zip_and_log`

Add return type annotation: `None`

(ANN202)